packages/expo-doctor/src/checks/checks.types.ts

```diff
@@ -0,0 +1,19 @@
+export interface DoctorCheck {
+  // description that will appear as each check is run
+  description: string;
+  runAsync: (params: DoctorCheckParams) => Promise<DoctorCheckResult>;
+}
+
+export interface DoctorCheckResult {
+  isSuccessful: boolean;
+  // many checks currently output their own issues, no need to duplicate
```

Comment:
```suggestion
  /** many checks currently output their own issues, no need to duplicate */
```
and related for doc blocks, makes it easier to read around the code.